Makefile

```diff
@@ -790,7 +790,7 @@ check-git-abc:
 	elif git -C "$(YOSYS_SRC)" submodule status abc 2>/dev/null | grep -q '^ '; then \
 	    echo "'abc' is a git submodule. Continuing."; \
 	    exit 0; \
-	elif [ -f "$(YOSYS_SRC)/abc/.gitcommit" ] && grep -q '\$$Format:%h\$$' "$(YOSYS_SRC)/abc/.gitcommit"; then \
+	elif ([ -f "$(YOSYS_SRC)/abc/.gitcommit" ] && grep -q '\$$Format:%h\$$' "$(YOSYS_SRC)/abc/.gitcommit") || [ -z "$(git submodule status abc | grep '^-' )" ]; then \
```

Comment:
-z or -n?

Reply:
-z

Reply:
Weird... on a fresh clone running `make check-git-abc` returns the expected error message. However, running `make`, still results in the error message seen in the issue statement.